rotkehlchen/externalapis/etherscan.py

```diff
@@ -285,11 +330,35 @@ def get_transactions(
             for entry in result:
                 gevent.sleep(0)
                 try:
+                    # Handle genesis block transactions
+                    if is_genesis_tx(entry):
+                        if self.db is None:
+                            raise DeserializationError('DB not initialized')
+                        dbtx = DBEthTx(self.db)
+                        trace_id = dbtx.get_max_trace_id(GENESIS) + 1
```

Comment:
Why you don't want to modify your `get_max_trace_id` method to accept tx hash type and not a pure `str` ? We should type things whenever is possible